### PR TITLE
minikube: upgrade to v0.28.2

### PR DESCRIPTION
**Pull request by tomerlange**
Fixed bootstrapper issue according to the comments of the previous PR.

### CODE REVIEW

**Comment by lebauce** [comment]
@tomerlange Cool. Could you also remove the 1.6.1 version dependency of "iptables" just like the previous PR ? Thanks

**Comment by tomerlange** [comment]
Sure. DONE

**Comment by lebauce** [comment]
add to whitelist

**Comment by lebauce** [comment]
@tomerlange nit: commit should begin with a prefix as much as possible, like : "ansible: cancel iptables downgrading".

Could you also cherry-pick this commit in your PR please : https://github.com/skydive-project/skydive/pull/1267/commits/f6b81fafc8e2e04e82983687e488cafefc05a64d ?

**Comment by lebauce** [comment]
run tests

**Comment by nplanel** [comment]
run tests

File: scripts/ci/install-minikube.sh [hunk 1]
```diff
@@ -101,8 +101,9 @@ stop() {
 start() {
         check_minikube
 
+	local args="--kubernetes-version $K8S_VERSION"
```
Comment (nplanel) [inline scripts/ci/install-minikube.sh]: could you fix the ident

**Comment by lebauce** [comment]
@tomerlange Could you also cherry-pick this commit in your PR please : f6b81fa ?

File: scripts/ci/install-minikube.sh [hunk 2]
```diff
@@ -6,13 +6,13 @@ OS=linux
...
-MINIKUBE_VERSION="v0.25.2"
+MINIKUBE_VERSION="v0.28.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
-KUBECTL_VERSION="v1.9.4"
-KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
+K8S_VERSION="v1.10.0"
+KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/$OS/$ARCH/kubectl"
 
-WITH_CALICO=true
```
Comment (nplanel) [inline scripts/ci/install-minikube.sh]: why disabling calico ?

**Comment by nplanel** [comment]
@tomerlange Could you also cherry-pick this commit in your PR please : f6b81fa ?

I can't see the cherry-picked commit

File: scripts/ci/install-minikube.sh (hunk 2 above)
Comment (tomerlange) [reply scripts/ci/install-minikube.sh]: Because there are issues when using Calico and Istio together.

**Comment by tomerlange** [comment]
Sorry, now cherry-picked

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

File: scripts/ci/install-minikube.sh (hunk 2 above)
Comment (nplanel) [reply scripts/ci/install-minikube.sh]: it's out of scope of this patch
please include this in the istio patch

File: scripts/ci/install-minikube.sh [hunk 3]
```diff
@@ -6,13 +6,13 @@ OS=linux
 ARCH=amd64
 TARGET_DIR=/usr/bin
 
-MINIKUBE_VERSION="v0.25.2"
+MINIKUBE_VERSION="v0.28.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
-KUBECTL_VERSION="v1.9.4"
-KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
+K8S_VERSION="v1.10.0"
```
Comment (nplanel) [inline scripts/ci/install-minikube.sh]: why not using the latest one v1.10.7 ?

**Comment by tomerlange** [comment]
run skydive-cdd-overview-tests

**Comment by tomerlange** [comment]
run skydive-ppc64le-tests

**Comment by tomerlange** [comment]
run skydive-ppc64le-tests

**Comment by tomerlange** [comment]
run skydive-k8s-tests

**Comment by tomerlange** [comment]
run skydive-k8s-tests

**Comment by tomerlange** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by tomerlange** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-scale-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**